### PR TITLE
Docs/guides 2.1 tag helpers

### PR DESCRIPTION
**Pull request by swilgosz**
Resolves: https://github.com/hanami/guides/issues/200

### CODE REVIEW

File: content/v2.1/helpers/overview.md [hunk 1]
```diff
@@ -3,8 +3,231 @@ title: Overview
...
+Given you have a view exposure defined:
+
+```ruby
+# app/views/books/show.rb
+
+module Bookshelf
+  module Views
+    module Books
+      class Show < Bookshelf::View
+        expose :fancy_header do
```
Comment (jodosha) [inline content/v2.1/helpers/overview.md:47]: @swilgosz Given parts are a new concept. Readers may have a hard time wrapping their heads around them.
For me, this is an advanced usage of Parts.

What if we replace `:fancy_header` with a simple `:book`?

```ruby
expose :book do
  Book.new(title: "Hanami")
end
```

Then the part will be about `Book`, not `FancyHeader`.

```ruby
def content
  helpers.tag.h1(value.title)
end
```

File: content/v2.1/helpers/overview.md [hunk 2]
```diff
@@ -3,8 +3,231 @@ title: Overview
...
+tag.p(class: "<script>alert()</script>")  # => <p class="&lt;script&gt;alert()&lt;/script&gt;"></p>
+```
+
+To bypass it use `html_safe` on the given content
+
+```ruby
+tag.p("<em>safe content</em>".html_safe)  # => <p><em>safe content</em></p>
+```
+
+### Standalone use hanami-view
```
Comment (jodosha) [inline content/v2.1/helpers/overview.md:156]: @swilgosz I'm for keeping out of the Guides any standalone usage of the Hanami gems.

Guides -> full Hanami app.
Docs -> standalone gems

File: content/v2.1/helpers/overview.md [hunk 3]
```diff
@@ -3,8 +3,231 @@ title: Overview
...
+  include Hanami::View::Helpers::TagHelper
+end
+
+class BaseView < Hanami::View
+  config.part_class = BasePart
+  config.scope_class = BaseScope
+end
+```
+
+## link_to helper
```
Comment (jodosha) [inline content/v2.1/helpers/overview.md:175]: @swilgosz What's your idea for organizing all the helpers' documentation? All in here or separated docs?

**Review by jodosha — COMMENTED** [review]
@swilgosz Thank you very much! I left a few inline comments. let's discuss. 🙂

File: content/v2.1/helpers/overview.md (hunk 1 above)
Comment (swilgosz) [reply content/v2.1/helpers/overview.md:47]: I like this simplification, thanks!

File: content/v2.1/helpers/overview.md (hunk 3 above)
Comment (swilgosz) [reply content/v2.1/helpers/overview.md:175]: At first I thought they'll fit in in a single document, but after working on form_for, that's not gonna happen for sure. I can extract to a separate doc, same as for others.

File: content/v2.1/helpers/overview.md (hunk 2 above)
Comment (swilgosz) [reply content/v2.1/helpers/overview.md]: Removed!

File: content/v2.1/helpers/overview.md (hunk 1 above)
Comment (swilgosz) [reply content/v2.1/helpers/overview.md]: Done!

File: content/v2.1/helpers/tag_helper.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+Then you can have the part with the helper used.
+
+```ruby
+# app/views/parts/book.rb
+
+module Bookshelf
+  module Views
+    module Parts
+      module Book
+        def content
```
Comment (jodosha) [inline content/v2.1/helpers/tag_helper.md:57]:
```suggestion
        def formatted_title
```

File: content/v2.1/helpers/tag_helper.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+  end
+end
+```
+
+Then in the template you can just access the part:
+
+```ruby
+# app/templates/books/show.html.erb
+
+<%= book.content %>
```
Comment (jodosha) [inline content/v2.1/helpers/tag_helper.md:71]:
```suggestion
<%= book.formatted_title %>
```

**Review by jodosha — APPROVED** [review]
@swilgosz Thank you!

**Pull request merged**